assets/sass/components/_button.scss

```diff
@@ -1,9 +1,13 @@
+@import '../mixins/focus';
+
 $extra-small-height: 16px;
 $small-height: 18px;
 $medium-height: 20px;
 $large-height: 24px;
 
 @mixin button-basics {
+	@include focus-with-background-and-color;
```

Comment:
I was going through this an looks good in general! I liked the way the two different versions work together. And Sass probably indeed works nicely here :) 

Just a question about the buttons. Now they seem to get a bit mixed styling. For instance on the front page of a course, the filter buttons  look like this:

![image](https://user-images.githubusercontent.com/43036333/90402128-aaec6200-e0a7-11ea-80b2-a3618fb5f778.png)

but then again the enroll-button looks like this:
![image](https://user-images.githubusercontent.com/43036333/90402235-d4a58900-e0a7-11ea-8f67-fc3c0a15d3fd.png)

What was the meant outlook for buttons?

Reply:
Hmm, yeah, I didn't touch the Bootstrap buttons because removing them from all student-facing things is on my to-do list for this week and it's not so easy to hook into the Bootstrap styles here.

Maybe this can be merged after that?

Reply:
Finalizing this pull request after the buttons sounds like a good plan.